src/components/CategoryFilter.js

```diff
@@ -1,10 +1,23 @@
 import React from "react";
 
-function CategoryFilter() {
+function CategoryFilter({categories, filterByCategory, isSelected}) {
+
+  const categoryButtons = categories.map(category => {
+    return (
+    <button
+      key={category}
+      onClick={filterByCategory}
+      className={category === isSelected ? "selected" : ""}
+      >
+      {category}
+    </button>
+    )
+  })
```

Comment:
It should look like this:
```suggestion
  const categoryButtons = categories.map(category => {
    return (
      <button
        key={category}
        onClick={filterByCategory}
        className={category === isSelected ? "selected" : ""}
      >
        {category}
      </button>
    )
  })
```